### ISSUE TITLE
Benchmark Compute.scala and nd4j

### ISSUE THREAD

**Issue by Atry** [ThoughtWorksInc/OpenCL.scala#60]
This will produce report for performance difference between Compute.scala and nd4j

**Comment by Atry** [comment]
Since Compute.scala is designed to perform complex computations in a batch, we should perform some common algorithms in both Compute.scala and nd4j.

- A CNN layer
- pooling
- BatchNormal
- softmax

**Comment by Atry** [comment]
Done
